proxy/grpcproxy/cache/store.go

```diff
@@ -73,7 +77,11 @@ func (c *cache) Add(req *pb.RangeRequest, resp *pb.RangeResponse) {
 	defer c.mu.Unlock()
 
 	if req.Revision > c.compactedRev {
-		c.lru.Add(key, resp)
+		if req.Revision == 0 {
+			c.lru.Set(key, resp, defaultCurrentTTL)
+		} else {
+			c.lru.Set(key, resp, defaultHistoricTTL)
+		}
```

Comment:
could you explain the rationale behind using defaultCurrentTTL when req.Revision == 0 and defaultHistoricTTL when req.Revision != 0?